sdk/monitor/Microsoft.Azure.Management.Monitor/AzSdk.RP.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>Insights_2018-03-01;Insights_2018-11-27-preview;insights_2019-03-01;insights_2017-04-01;insights_2015-04-01;insights_2016-03-01;insights_2017-05-01-preview;insights_2017-11-01-preview;insights_2018-01-01;insights_2017-12-01-preview;insights_2018-04-16;insights_2019-10-17-preview;insights_2021-04-01;insights_2021-09-01;insights_2022-02-01;insights_2022-06-01</AzureApiTag>
+    <AzureApiTag>Insights_2022-06-01;Insights_2020-10-01;Insights_2015-04-01;Insights_2016-03-01;Insights_2019-03-01;Insights_2021-09-01-preview;Insights_2017-05-01-preview;Insights_2018-03-01;Insights_2018-01-01;Insights_2019-10-17-preview;Insights_2018-04-16;Insights_2018-11-27-preview;insights_2017-12-01-preview;</AzureApiTag>
```

Comment:
This looks like unrelated change. Is this intended?

Reply:
Hi @vishweshbankwar,

I have no idea.  I just change the dependency value for OpenTelemetry, launch the build and the tests like it's describe in the contributing et voilà

Reply:
Ok - please revert this change and we can merge then. Thanks!

Reply:
@vishweshbankwar done 👍